scripts/dev_setup.sh

```diff
@@ -626,6 +626,21 @@ function install_python3 {
   fi
 }
 
+function install_postgres {
+  if [[ "$PACKAGE_MANAGER" == "apt-get" ]] || [[ "$PACKAGE_MANAGER" == "apk" ]]; then
+    install_pkg libpq-dev "$PACKAGE_MANAGER"
+  fi
+  if [[ "$PACKAGE_MANAGER" == "pacman" ]] || [[ "$PACKAGE_MANAGER" == "yum" ]]; then
+    install_pkg postgresql-libs "$PACKAGE_MANAGER"
+  fi
+  if [[ "$PACKAGE_MANAGER" == "dnf" ]]; then
+    install_pkg libpq-devel "$PACKAGE_MANAGER"
+  fi
```

Comment:
even though I use pacman / arch, it is probably time we kill things we aren't evaluating in CI or on a regular basis. this is a disaster!!

Reply:
potentially yes, but outside scope of this PR :-)